Sources/SourceKitLSPAPI/BuildDescription.swift

```diff
@@ -2,23 +2,26 @@
 //
 // This source file is part of the Swift open source project
 //
-// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Copyright (c) 2023-2024 Apple Inc. and the Swift project authors
 // Licensed under Apache License v2.0 with Runtime Library Exception
 //
 // See http://swift.org/LICENSE.txt for license information
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 //===----------------------------------------------------------------------===//
 
-import Foundation
+import struct Foundation.URL
 
-/*private*/ import struct Basics.AbsolutePath
-/*private*/ import func Basics.resolveSymlinks
-// FIXME: should not import this module
-import Build
-// FIXME: should be internal imports
-import PackageGraph
-/*private*/ import SPMBuildCore
+private import struct Basics.AbsolutePath
+private import func Basics.resolveSymlinks
+
+private import SPMBuildCore
+
+// FIXME: should import these module with `private` or `internal` access control
+import class Build.BuildPlan
```

Comment:
Unfortunately, these types were directly exposed to SourceKit-LSP. We need to come with more constrained wrappers for them in the future.